components/DropDown/index.jsx

```diff
@@ -0,0 +1,115 @@
+import React, {PropTypes, Component} from 'react';
+
+import styles from './index.scss';
+
+import {
+  clearfix
+} from '../index.scss';
+
+export default class Dropdown extends Component {
+  static propTypes = {
+    items: PropTypes.array.isRequired,
```

Comment:
We could specify it is an array of objects. And possibly specify the shape of the objects themselves ^^